contracts/Base_SpokePool.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
This is an exact copy of Optimism_SpokePool.sol, only the name has been changed.